src/linux_network.py

```diff
@@ -2,77 +2,153 @@
 #
 # igcollect - Linux network
 #
-# Copyright (c) 2016, InnoGames GmbH
+# Copyright (c) 2018, InnoGames GmbH
 #
 
 from argparse import ArgumentParser
 from time import time
+import os
+
+
+class InterfaceStatistics(object):
+    '''
+        Supported types of interfaces for a metrics sending
+        If there is more than one conditions they multipied by AND
+    '''
```

Comment:
PEP8 says to use 3 double quotes on docstrings and the content should not be indented.